src/DynamoCore/Engine/CodeCompletion/CodeCompletionServices.cs

```diff
@@ -191,7 +191,7 @@ private void AddTypesToCompletionData(string stringToComplete, List<CompletionDa
             var partialName = stringToComplete.ToLower();
             // Add matching Classes
             var classMirrorGroups = StaticMirror.GetAllTypes(core).
-                Where(x => !x.IsHiddenInLibrary && x.Alias.ToLower().Contains(partialName)).
+                Where(x => x.Alias.ToLower().Contains(partialName)).
```

Comment:
We should compute shortest names in `Symbol.GetShortestUniqueNames` while including hidden classes and later filter out the hidden class names so that they are not displayed to the user.

Reply:
I'm pretty confused by this change. Can you tell me how it affects the following use cases.

1. Previously I wrote some code like `__Apply` or some other hidden class, like `ElementResolver.GetElement` in the revit elements namespace. What happens to my code block?

2. If I try to access those hidden types now, what happens?

Reply:
You can still access those hidden types from code block nodes. They just don't show up in the library or autocomplete. My fixes here are to ensure they also don't show up on doing node to code operations.